code/modules/shuttle/on_move.dm

```diff
@@ -106,6 +106,7 @@ All ShuttleMove procs go here
 		return
 
 	loc = newT
+	SEND_SIGNAL(src, COMSIG_MOVABLE_LOCATION_CHANGE, oldT)
```

Comment:
Why not forceMove?

Reply:
im guessing its so that the movements cant have side effects? i guess ninjanomnom would know

Reply:
uses abstract_move now